test/e2e/framework/network_pods.go

```diff
@@ -325,6 +325,7 @@ func (np *NetworkPod) buildTCPCheckConnectorPod() {
 					Command: []string{
 						"sh",
 						"-c",
+						//nolint:dupword // Ignore duplicate word "in"
 						"for in in $(seq 1 $BUFS_NUM);" +
```

Comment:
This looks like a typo:

```suggestion
						"for i in $(seq $BUFS_NUM);" +
```

None of the loop variables are used so their name doesn’t matter. (Also drive-by fixing the `seq` command to match line 331.)